COLLADASaxFrameworkLoader/src/COLLADASaxFWLMeshLoader.cpp

```diff
@@ -913,11 +920,13 @@ namespace COLLADASaxFWL
                 mColorList.push_back ( color );
             }
         }
+		return false;
     }
 
     //------------------------------
-    void MeshLoader::initializeTexCoordsOffset ()
+    bool MeshLoader::initializeTexCoordsOffset ()
     {
+		bool has_errors = false;
```

Comment:
yes, not used. That was a left over that i actually never needed. I tried to change the pull request but i could not find out how to do that. Shall i close it and create a new one? Or if its otherwise OK, you could just remove that line?